libraries/chain/wasm_interface.cpp

```diff
@@ -689,8 +724,8 @@ class console_api : public context_aware_api {
    public:
       using context_aware_api::context_aware_api;
 
-      void prints(null_terminated_ptr str) {
-         context.console_append<const char*>(str);
+      void prints(const char *str) {
+         context.console_append(str);
       }
```

Comment:
Same comment as above, how does this check for null?

Reply:
It will, when I change it back. Again, not sure what I did to get git to do this wrong.